numpy/core/tests/test_simd.py

```diff
@@ -0,0 +1,422 @@
+# NOTE: Please avoid the use of numpy.testing since NPYV intrinsics
+# may be involved in their functionality.
+import pytest
+from numpy.core._simd import targets
+
+class _Test_Utility:
+    # submodule of the desired SIMD extention, e.g. targets["AVX512F"]
+    npyv = None
+    # the current data type suffix e.g. 's8'
+    sfx  = None
```

Comment:
I think this would make more sense as an object rather than a base class:
```suggestion
class SuffixInfo:
    def __init__(self, npyv, sfx):
        self.sfx = sfx  # the current data type suffix e.g. 's8'
        self.npyv = npyv  # submodule of the desired SIMD extention, e.g. targets["AVX512F"]

    def __getattr__(self, attr):
        """
        To call NPV intrinsics without the prefix 'npyv_' and
        auto suffixing intrinsics according to class attribute 'sfx'
        """
        return getattr(self.npyv, attr + "_" + self.sfx)
```

Reply:
`_Test_Utility` is sharing common functions that affected by the tested vector data type and SIMD extension module